src/fairseq2/models/wav2vec2/_config.py

```diff
@@ -201,14 +203,17 @@ def validate(self) -> None:
 
 
 def register_wav2vec2_configs(context: RuntimeContext) -> None:
-    registry = context.get_config_registry(Wav2Vec2Config)
-
-    arch = registry.decorator
+    arch = context.get_config_registry(Wav2Vec2Config).decorator
```

Comment:
we can still keep all configs on fs2 side

Reply:
Why? The open-source recipes do not use them and they are mms-specific. The only configs from the open-source SSL recipe that I found were `base_960h` and `large_960h`.

Each of these functions will be a standalone `.yaml` file after the migration and this might be a good change to prune them for the ones that are relevant and add them to the `mms`-specific SSL recipe.